pyproject.toml

```diff
@@ -0,0 +1,99 @@
+[build-system]
+requires = [
+  "setuptools >= 61",
+  "setuptools_scm[toml] >= 8",
+]
+build-backend = 'setuptools.build_meta'
+
+[project]
+name="DC_OCEAN"
+dynamic = ["version"]
+description = "DC_OCEAN: An algorithm to detect the ocean in-situ duplicate profiles (Song et al., 2024, FMS)"
+readme = "README.md"
+authors = [
+  {name = "Zhetao Tan", email = "tanzhetao@mail.iap.ac.cn"},
+  {name = "Xinyi Song", email = "songxinyi231@mails.ucas.ac.cn"},
+  {name = "Lijing Cheng"},
+]
+maintainers = [
+  {name = "Zhetao Tan", email = "tanzhetao@mail.iap.ac.cn"},
+]
+keywords = ["oceanography", "duplicate", "quality control"]
+requires-python = ">= 3.7"
```

Comment:
@zqtzt , consider moving this to `3.10`, or if you have a good reason for not, at least `3.8`.

Reply:
Yes, we can move it to '>= 3.8'